### PR TITLE
zh-cn: revise translation of "Cross browser testing"

### PR DESCRIPTION
**Pull request by hanyujie2002**
relates to #19921

### CODE REVIEW

File: files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
 - [处理常见的 HTML 和 CSS 问题](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/HTML_and_CSS)
-  - : 现在，我们着重考虑你在 HTML 和 CSS 代码中可能遇到的一些常见的跨浏览器问题，以及哪些工具可以用来防止问题发生和解决问题。这包括代码检查，处理 CSS 前缀，使用浏览器开发工具找出问题，使用 polyfill 添加浏览器支持，解决响应式问题等。
+  - : 现在，我们着重考虑你在 HTML 和 CSS 代码中可能遇到的一些常见的跨浏览器问题，以及哪些工具可以用来防止问题发生和解决问题。这包括代码检查，处理 CSS 前缀，使用浏览器开发工具找出问题，使用 polyfill 添加浏览器支持以及解决响应式（responsive）设计问题等。
 - [处理常见的 JavaScript 问题](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/JavaScript)
-  - : 现在我们来看看常见的跨浏览器 JavaScript 问题，以及怎么解决它们。这会包括使用浏览器开发工具跟踪和解决问题的信息，使用 polyfill 和库来解决问题，让现代的 JavaScript 特性在老浏览器中工作等等。
+  - : 现在我们将研究常见的跨浏览器 JavaScript 问题以及如何解决它们。这包括使用浏览器开发工具来追踪和修复问题，使用 polyfills 和库来解决问题，以及在旧版浏览器中实现现代 JavaScript 功能等内容。
 - [处理常见的无障碍问题](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility)
-  - : 然后，我们把注意力转向无障碍上，提供常见问题的信息，怎么做简单的测试，和怎么使用审计/自动化工具找到无障碍问题。
+  - : 然后，我们把注意力转向无障碍上，提供有关常见问题的信息，介绍如何进行简单的测试，以及如何使用审计/自动化工具发现无障碍问题。
 - [实现特性检查](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection)
```
Comment (hanyujie2002) [inline files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md:28]:
```suggestion
- [实现特性检测](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection)
```

File: files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
 title: 跨浏览器测试
 slug: Learn/Tools_and_testing/Cross_browser_testing
+l10n:
+  sourceCommit: 26e2f9883e0e73def04c0e86fec6da3ec42e66b3
 ---
 
 {{LearnSidebar}}
 
-本模块的重点是在不同的浏览器上测试 web 项目。我们研究确定你的目标受众（例如，你最需要担心哪些用户、浏览器和设备），如何去做测试，不同类型的代码中会面临的主要问题以及如何缓解这些问题，哪些工具在帮助你测试和解决问题方面最有用，以及如何使用自动化来加速测试。
+本模块主要介绍如何进行跨浏览器测试 Web 项目。我们将探讨如何确定目标用户群（例如，你应该关注哪些用户、浏览器和设备），进行测试的方法，不同类型代码可能遇到的主要问题及其解决策略，哪些工具在测试和问题解决过程中最为有效，以及如何利用自动化技术加快测试进程。
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md:10]: “进行”后面要跟宾语，“如何进行……测试……项目”是病句。

```suggestion
本模块主要介绍如何跨浏览器测试 Web 项目。我们将探讨如何确定目标用户群（例如，你应该关注哪些用户、浏览器和设备）、进行测试的方法、不同类型代码可能遇到的主要问题及其解决策略、哪些工具在测试和问题解决过程中最为有效，以及如何利用自动化技术加快测试进程。
```

File: files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
 ## 前提
 
-在你尝试使用这里详细介绍的工具时，你需要已经学习核心 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 语言。
+在你尝试使用这里详细介绍的工具前，你需要掌握 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 语言的基本知识。
 
 ## 指南
 
 - [跨浏览器测试简介](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/Introduction)
-  - : 本文通过提供跨浏览器测试主题的概述，回答诸如“什么是跨浏览器测试？”、“将遇到的最常见类型的问题是什么？”和“什么是测试，识别和解决问题的主要方法？”等问题。
+  - : 本文是本模块的起始点，通过概述跨浏览器测试主题，回答了诸如“什么是跨浏览器测试？”、“会遇到哪些常见的问题？”和“测试，识别和解决问题主要有哪些方法？”等问题。
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md:19]:
```suggestion
  - : 本文是本模块的起始点，通过概述跨浏览器测试主题，回答了诸如“什么是跨浏览器测试？”、“会遇到哪些常见的问题？”和“测试、识别和解决问题主要有哪些方法？”等问题。
```

File: files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
 - [测试策略](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/Testing_strategies)
-  - : 接下来，我们深入了解测试，寻找目标受众（例如：你需要确定测试的浏览器，设备和其他部分），低成本测试策略（必要时对各种设备和一些虚拟机进行 adhoc 测试），更高科技策略（自动化，使用专用测试 app）和用户组测试。
+- : 接下来，我们将深入探讨测试流程，确定目标受众（例如：你需要明确哪些浏览器、设备等需要进行测试），实施低成本测试策略（在必要时在各种实体设备和虚拟机上进行临时测试），采用更高级的技术策略（例如自动化测试和使用专门的测试应用程序）以及进行用户群体测试。
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md:21]:
```suggestion
  - : 接下来，我们将深入探讨测试流程、确定目标受众（例如：你需要明确哪些浏览器、设备等需要进行测试）、实施低成本测试策略（在必要时在各种实体设备和虚拟机上进行临时测试）、采用更高级的技术策略（例如自动化测试和使用专门的测试应用程序）以及进行用户群体测试。
```

File: files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md [hunk 5]
```diff
@@ -1,31 +1,33 @@
...
 - [处理常见的 HTML 和 CSS 问题](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/HTML_and_CSS)
-  - : 现在，我们着重考虑你在 HTML 和 CSS 代码中可能遇到的一些常见的跨浏览器问题，以及哪些工具可以用来防止问题发生和解决问题。这包括代码检查，处理 CSS 前缀，使用浏览器开发工具找出问题，使用 polyfill 添加浏览器支持，解决响应式问题等。
+  - : 现在，我们着重考虑你在 HTML 和 CSS 代码中可能遇到的一些常见的跨浏览器问题，以及哪些工具可以用来防止问题发生和解决问题。这包括代码检查，处理 CSS 前缀，使用浏览器开发工具找出问题，使用 polyfill 添加浏览器支持以及解决响应式（responsive）设计问题等。
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md:23]:
```suggestion
  - : 现在，我们着重考虑你在 HTML 和 CSS 代码中可能遇到的一些常见的跨浏览器问题，以及哪些工具可以用来防止问题发生和解决问题。这包括代码检查、处理 CSS 前缀、使用浏览器开发工具找出问题、使用 polyfill 添加浏览器支持以及解决响应式（responsive）设计问题等。
```

File: files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md [hunk 6]
```diff
@@ -1,31 +1,33 @@
...
 ## 前提
 
-在你尝试使用这里详细介绍的工具时，你需要已经学习核心 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 语言。
+在你尝试使用这里详细介绍的工具前，你需要掌握 [HTML](/zh-CN/docs/Learn/HTML)、[CSS](/zh-CN/docs/Learn/CSS) 和 [JavaScript](/zh-CN/docs/Learn/JavaScript) 语言的基本知识。
 
 ## 指南
 
 - [跨浏览器测试简介](/zh-CN/docs/Learn/Tools_and_testing/Cross_browser_testing/Introduction)
-  - : 本文通过提供跨浏览器测试主题的概述，回答诸如“什么是跨浏览器测试？”、“将遇到的最常见类型的问题是什么？”和“什么是测试，识别和解决问题的主要方法？”等问题。
+  - : 本文是本模块的起始点，通过概述跨浏览器测试主题，回答了诸如“什么是跨浏览器测试？”、“会遇到哪些常见的问题？”和“测试、识别和解决问题主要有哪些方法？”等问题。
```
Comment (hanyujie2002) [inline files/zh-cn/learn/tools_and_testing/cross_browser_testing/index.md:19]:
```suggestion
  - : 本文是本模块的起始点，通过概述跨浏览器测试主题，回答了诸如什么是跨浏览器测试、会遇到哪些常见的问题和测试以及识别和解决问题主要有哪些方法等问题。
```

**Review by yin1999 — APPROVED** [review]
Thank you @hanyujie2002

**Pull request merged**